tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
   "extends": "@foxglove/tsconfig/base",
-  "include": ["./src/**/*"],
+  "include": ["./src/**/*", "*.d.ts"],
   "compilerOptions": {
     "rootDir": "./src",
     "outDir": "./dist/esm",
     "lib": ["es2020", "dom"],
+    "importHelpers": false,
```

Comment:
I suggest breaking this out into a separate PR (let's discuss it on a PR in https://github.com/foxglove/tsconfig rather than overriding our own default in just one library).

Setting `importHelpers: true` was done by @jhurliman in https://github.com/foxglove/studio/pull/1100, with the justification:

> Our base tsconfig disabled require()ing tslib in favor of including it directly into output code. This is a cruel thing to do to anyone who tries to use one of our libraries in their own projects, and it's the reason our output main.js has 12 separate copies of tslib baked into it. We now follow the default tsc behavior around tslib usage

I buy that argument within the studio repo, but it's less clear to me what is the best practice for libraries. We should wait until John is back online before making a decision.

Reply:
I don't understand that justification - I found the opposite. With importHelpers: true, the dist code imports "tslib" and our package does not depend on tslib - so it is unusable when installed. I believe this is true of many of our packages if you try using them without typescript.

Reply:
yeah - we should make a decision here and either add tslib as a dependency to all packages or set this to false in our base config. I lean towards setting this to false and making the packages self-contained without dependencies, but should let john make his case.